axiom/tests/test_txn.c

```diff
@@ -6725,7 +6750,7 @@ static void test_txn_accept_distributed_trace_payload_w3c(void) {
   TEST_TXN_ACCEPT_DT_PAYLOAD_RESET
   nr_distributed_trace_destroy(&txn.distributed_trace);
   txn.distributed_trace = nr_distributed_trace_create();
-  nr_distributed_trace_set_trace_id(txn.distributed_trace, "35ff77");
+  nr_distributed_trace_set_trace_id(txn.distributed_trace, "35ff77", false);
```

Comment:
This test should have two versions for two possible values of `nrtxnopt_t.distributed_tracing_pad_trace_id`.

Reply:
Further evaluation revealed that it is unimportant what is the value for the third argument of `nr_distributed_trace_set_trace_id` function. See f71fca4a2ba3fdb2f17d5816ef3f9abc63f3cdd7 which provides explanations why.